app/src/main/java/com/pmatuki/wowpapers/application/WowpapersApplication.kt

```diff
@@ -1,11 +1,14 @@
 package com.pmatuki.wowpapers.application
 
 import android.app.Application
+import javax.inject.Inject
 
-class WowpapersApplication: Application() {
+class WowpapersApplication @Inject constructor() : Application() {
```

Comment:
You probably shouldn't inject the constructor of Application class once you won't inject it anywhere else. I don't know exactly how ToothPick works, but probably you need to have a function here that pass this class as a parameter to the application context be added in the graph, not annotating it.

Reply:
Done!